### ISSUE TITLE
Admin | Dashboard | Ability to view PII

### ISSUE THREAD

**Issue by nikitarockz** [department-of-veterans-affairs/caseflow-feedback#165]
As a Product Support Analyst, I want to be able to see the PII that the users has entered when submitting a feedback, so that I can process the support tickets without getting into a confrontation with the user by asking again to give me the PII that they have already provided in the User Feedback form or missing  out on the opportunity because the user is no longer working that case/file and can't remember.

# Acceptance Criteria
1.  Verify that there is a new column "PII Information" added to the right of the "Feedback" column. 
1. Verify that the new "PII Information" column displays the data from the "If you are having an issue with a specific case number or Veteran ID, please let us know. It's safe to put PII in this field." field of the User Feedback form.

# Mocks
**_Admin Feedback page - new PII column_**
![image](https://user-images.githubusercontent.com/23063044/37659356-1d84243a-2c26-11e8-9cab-1104787eb205.png)

**_User Feedback Form -  Pull data from this field_**
![image](https://user-images.githubusercontent.com/23063044/35820313-382f79da-0a73-11e8-999a-96fd4deefda2.png)

# Related Stories
**_PM Issues_**
Caseflow Feedback | Create Engineering stories to Modify UI and functionality [#1767](https://github.com/department-of-veterans-affairs/appeals-pm/issues/1767)

**Comment by Chingujo** [comment]
@amprokop @nikitarockz 
tldr: The PII field was added to the front facing user form but does not appear on the table. Could be a situation where we added it on the feedback ticket and not on the table or a bug where it's not showing on the admin table. 

### Problem 

Currently when a user fills out the form on the ticket, there is a box that allows them to enter PII. However, since this information does not show on the table, it means support has to spend time reaching out to the user for information that was a _already provided_. It also delays troubleshooting.
Not sure how long this has persisted but +1 on Alex's question to see on the duration it took to surface. 

### Current table
![feedback](https://user-images.githubusercontent.com/23316107/37668513-12e719be-2c3b-11e8-92e7-6f6c818bcd14.png)
